### PR TITLE
Pass error message to custom component as string

### LINKED ISSUE #11: CustomField should pass error as string
The CustomField component currently passes a div with classes and aria tags. Idealy it should only pass a string and lett the customComponent handle the error. 

In stead of making an inlineError node:
`const inlineError = createInlineError(name, inlineErrorClass, meta);`
And passing that to the custom component. Only the error message from `meta` should be passed.

### PR DESCRIPTION
**Pull request by MartinMekk**
Fixes #11

### CODE REVIEW

File: example/login.js [hunk 1]
```diff
@@ -6,13 +6,13 @@ import CustomField from '../src/fields/custom-field';
...
 
-function MyCustomComponent({ children, input, inlineError, meta, ...props }) {
+function MyCustomComponent({ children, input, errorMessage, meta, ...props }) {
     const id = ('' + Math.random()).slice(2);
     return (
         <div>
             <label htmlFor={id}>{children}</label>
             <input type="text" id={id} {...input} {...props} />
-            <div>{inlineError}</div>
+            <div role="alert" aria-live="assertive">{errorMessage}</div>
```
Comment (nutgaard) [inline example/login.js:15]: I feel like we should somehow force the use of aria-attributes here. But not sure how we can acheive it. :/

File: src/fields/custom-field.js [hunk 2]
```diff
@@ -1,19 +1,19 @@
 import React, { PropTypes as PT, cloneElement } from 'react';
 import { Field } from 'redux-form';
-import { createInlineError, fieldClasses } from './field-utils';
+import { fieldClasses } from './field-utils';
```
Comment (nutgaard) [inline src/fields/custom-field.js:3]: Is `createInlineError` used elsewhere, or can we delete it?

Comment (MartinMekk) [reply src/fields/custom-field.js:3]: Not used in custom field. It is used in the labeled field component though. But the labeled field component handles everything for the user. While the custom component lets the users handle everything themselves.

File: example/login.js (hunk 1 above)
Comment (MartinMekk) [reply example/login.js:15]: I think its ok to let the user handle everything here. Thats the purpose of the component. If you want to let the component handle things for you, then you can use the labeled field. Well, that's the idea anyways...

**Review by nutgaard — APPROVED** [review]

**Pull request merged**